jtune.py

```diff
@@ -1753,6 +1756,41 @@ def _at_exit(raw_gc_log=None, jmap_data=None, jstat_data=None,
                   optimized_for_ygcs_rate)
 
 
+def _generate_rotated_gc_log_name(gc_log_file, log_number):
+    return gc_log_file + "." + str(log_number)
```

Comment:
I use str.format consistently in jtune, lets do it the same for this line (even after we remove the function):

return "{0}.{1}".format(gc_log_file, log_number)